rust/processor/src/bq_analytics/gcs_handler.rs

```diff
@@ -22,6 +21,7 @@ pub async fn upload_parquet_to_gcs(
     file_path: &PathBuf,
     table_name: &str,
     bucket_name: &str,
+    bucket_root: &str,
```

Comment:
Let's make this a Path / PathBuf if possible, less room for error. For example, if someone starts / ends this with a slash it'll cause issues.

Reply:
good call!